contents/blog/testing-in-production.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Testing in production: what it is, why we do it, and how"
```

Comment:
I'm just looking at the Google results for this and trying to figure out a headline that stands out. 

![Screenshot 2023-06-30 at 15 08 19](https://github.com/PostHog/posthog.com/assets/92976667/3d132119-58a6-4304-9030-6210d1ed87df)

Random thoughts:

- Why we test in production (and you should to)
- Good engineers test. Great engineers test in production (probably a bit much)
- Why modern engineers test in production

Reply:
I like Why we test in production (and you should to)